config/prowgen/prowspecs/specs.go

```diff
@@ -177,7 +190,13 @@ func (m *BranchSpec) GenerateJobFile() *pkg.JobFile {
 		m.prowContext.Periodics(pkg.E2ETestFeatureGatesDisabled(m.prowContext, kubernetesVersion, m.e2eCPURequest, m.e2eMemoryRequest), 24)
 	}
 
-	for _, container := range []string{"controller", "acmesolver", "ctl", "cainjector", "webhook"} {
+	// Apply the default set of container names if none have been specified
+	// TODO: this is the set that we want to migrate to in the future
+	if m.containerNames == nil {
+		m.containerNames = []string{"controller", "acmesolver", "startupapicheck", "cainjector", "webhook"}
+	}
```

Comment:
nitpick: Do you think it'd be easier to edit this in the future if the `master` branch in `config/prowgen/prowspecs/specs.go` also specified its container names in that file, rather than defaulting here? I'm fine with the defaulting, but it's a little jarring to see `master` be a special case.

Reply:
The goal is that v1.15, 1.16 etc. will also not have this option defined.
So it gradually becomes unused and can be removed in the (far) future.